src/navigator/DrawerTopBar.tsx

```diff
@@ -43,9 +45,14 @@ function DrawerTopBar({ middleElement, rightElement, scrollPosition, testID }: P
 
   return (
     <Animated.View testID={testID} style={viewStyle}>
-      <TouchableOpacity style={styles.hamburger} onPress={onPressHamburger} hitSlop={iconHitslop}>
-        <Hamburger />
-      </TouchableOpacity>
+      <View style={styles.hamburger}>
+        <View style={styles.leftElement}>
```

Comment:
I think you should be able to combine these two into a single view and drop the hamburger naming, since its no longer only the hamburger